package.json

```diff
@@ -36,13 +36,13 @@
   "license": "W3C-20150513",
   "dependencies": {
     "@tpluscode/rdf-string": "^0.2.27",
-    "@types/rdf-js": "^4.0.2",
     "array-permutation": "^0.2.0",
     "n3": "^1.16.3",
     "rdf-js": "^4.0.2",
     "yaml": "^2.2.1"
   },
   "devDependencies": {
+    "@rdfjs/types": "^1.1.0",
```

Comment:
This should be in your dependencies or peer dependencies since some of the types are exposed to the package consumer.